deployment/helm/ditto/values.yaml

```diff
@@ -169,8 +169,15 @@ dbconfig:
 ingress:
   # enabled whether Ingress should be enabled as alternative to the contained nginx
   enabled: false
+  # namespace for ingress controller, must exist
+  namespace: ingress-nginx
+  # Ingress-NGINX version
+  nginxIngressVersion: v1.8.0
+  # Nginx Version
+  nginxVersion: "1.21.6"
```

Comment:
As suggested already, please move that under a new `controller` section as part of the `ingress`.
And add a flag whether to deploy the ingress-controller or not - by default it is probably ok to deploy it when `ingress.enabled = true`.

Reply:
The configuration moved to controller section.